src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInPage.java

```diff
@@ -170,4 +228,15 @@ private String urlWithInstantGlobals(String[] instantGlobals, Page page) {
   public void logTrackingCookieValue() {
     Log.info("Geo cookie: ", driver.manage().getCookieNamed("Geo").getValue());
   }
+
+  private void isConsentCookieSetToAccepted() {
+    final String cookieValue = driver.manage().getCookieNamed("euconsent").getValue();
```

Comment:
If you extract the part in which you get cookie value and then its length to a separate method (for example `getEuConsentCookieLength()`), you'll  be able to re-use it in `isConsentCookieSetToAccepted()` and `isConsentCookieSetToRejected()` and make them shorter.